app/services/processors/blog_metrics_updater.rb

```diff
@@ -0,0 +1,20 @@
+module Processors
+  class BlogMetricsUpdater < BaseService
+    def call
+      update_blog_post_visits_metrics
+      update_technologies_visits_metrics
+    end
+
+    private
+
+    def update_blog_post_visits_metrics
+      BlogPost.find_each(batch_size: 50) do |blog_post|
```

Comment:
You could add lazy here to improve the perfomance of the query, it adds a limit to the query.

Reply:
Where exactly?
`BlogPost.lazy.find_each(batch_size: 50)` throws a `NoMethodError`
`BlogPost.find_each(batch_size: 50).lazy` seems to have no impact on the DB, thus making the tests fail
I didn't know that method, so I'm not sure how to use it 😅

Reply:
It should be something like this: `BlogPost.find_each(batch_size: 50).lazy.each`

Reply:
All right, I'll see why it's making the tests fail then. Thanks!